Sources/XMTPiOS/Frames/FramesClient.swift

```diff
@@ -0,0 +1,101 @@
+//
+//  File.swift
+//  
+//
+//  Created by Alex Risch on 3/28/24.
+//
+
+import Foundation
+import LibXMTP
+
+public typealias FrameActionBody = Xmtp_MessageContents_FrameActionBody
+public typealias FrameAction = Xmtp_MessageContents_FrameAction
+
+class FramesClient {
+    var xmtpClient: Client
+    var proxy: OpenFramesProxy
+
+    init(xmtpClient: Client, proxy: OpenFramesProxy? = nil) {
+        self.xmtpClient = xmtpClient
+        self.proxy = proxy ?? OpenFramesProxy()
+    }
+
+    func signFrameAction(inputs: FrameActionInputs) async throws -> FramePostPayload {
```

Comment:
I think you need to add public on the front of this for it actually be accessible.
```suggestion
    public func signFrameAction(inputs: FrameActionInputs) async throws -> FramePostPayload {
```